modules/ballerina-native/src/main/ballerina/ballerina/net/httpsession/natives.bal

```diff
@@ -0,0 +1,77 @@
+package ballerina.net.httpsession;
```

Comment:
put this file in the ballerina.net.http package.  You can name the file as session.bal